docs/README.md

```diff
@@ -1,24 +1,40 @@
-# Welcome to the Cosmos Docs!
+# Welcome to the SDK Docs!
+
+Welcome to the SDK docs!
+
 
 ![cosmonaut reading the cosmos docs in space](./cosmos-docs.jpg)
 
-Cosmos is a decentralized network of independent parallel blockchains, each powered by classical BFT consensus algorithms like Tendermint.
+## Learn the SDK
+
+### SDK Intro
+
+If you are a newcomer and would like to learn more about the Cosmos SDK, this **[intro](./sdk.md)** is a good starting place.
+
+### SDK tutorial
+
+If you like to learn by doing, you can follow the **[SDK application tutorial](https://github.com/cosmos/sdk-application-tutorial)**. It showcases how to build an SDK-based blockchain from scratch, and teaches you about the basic principles the SDK in the process. 
+
+## Use the SDK
+
+The following sections contain the information you need if you want to build a fully-fonctional SDK-based blockchain:
+
+>*NOTE*: We are currently working on improving the docs. Some info might be missing. 
```

Comment:
i would remove this and link to either the forum (e.g., for clarification) or issues (for  outright wrong stuff or request for missing information)